extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -236,6 +232,9 @@ export class SystemLayer {
 
     /** @private {?number|?string} */
     this.timeoutId_ = null;
+
+    /** @private {?number|?string} */
+    this.storyHasNewPageTimeoutId_ = null;
```

Comment:
I think we want to use the same `timeoutId_` for all the messages that can be displayed in the system layer, otherwise they'd just be displayed on top of each other (and that'll make the code easier).